cdap-docs/developers-manual/source/building-blocks/streams.rst

```diff
@@ -63,12 +63,16 @@ changed, using the :ref:`http-restful-api-stream`, the :ref:`stream-client` of t
 :ref:`client-api`, or by using the :ref:`Command Line Interface. <cli>`
 
 
-.. rubric::  Truncating a Stream
+.. rubric::  Truncating and Deleting a Stream
 
 Streams can be truncated, which means deleting all events that were ever written to the
 Stream. This is permanent and cannot be undone. They can be truncated through the using
```

Comment:
I don't think you're supposed to capitalize words in a paragraph just because they appear in the title of the paragraph. If that was the case, then why don't we capitalize "truncating" and "deleting"?